packages/d/drumstick/package.yml

```diff
@@ -1,8 +1,8 @@
 name       : drumstick
-version    : 2.9.0
-release    : 14
+version    : 2.9.1
+release    : 15
 source     :
-    - https://sourceforge.net/projects/drumstick/files/2.9.0/drumstick-2.9.0.tar.bz2 : a7437c11e0ad5443c21b33f0891c4f748d574f95d0e6970a6040c95db6184eb3
+    - https://sourceforge.net/projects/drumstick/files/2.9.1/drumstick-2.9.1.tar.gz/download : a7049333d5411faf4d91a81ae666746106b578897fc0713a84325f65fdd06ffb
```

Comment:
According to the `package.yml` docs, the trailing `/download` in the URL should be removed.